packages/wrangler/src/config/validation.ts

```diff
@@ -3172,6 +3178,14 @@ const validateMigrations: ValidatorFn = (diagnostics, field, value) => {
 				"string"
 			) && valid;
 
+		valid =
```

Comment:
Presumably we also need to validate that every string in `new_sqlite_classes` matches an existing durable object name?

Reply:
This is validated in EWC. I am copying the existing validation that is present for new_classes so that they have the same behavior.